applications/resturant/tests/test_ssl.py

```
@@ -0,0 +1,55 @@
+import json
+from django.test import TestCase, RequestFactory
+from django.http import JsonResponse
+
+
+# Simple test-only view to check secure status
+def secure_check_view(request):
```

Ruff (0.11.9)

Missing return type annotation for public function `secure_check_view`

(ANN201)